test/test_file_blob.rb

```diff
@@ -491,9 +491,6 @@ def test_vendored
     # Carthage
     assert sample_blob('Carthage/blah').vendored?
 
-    # Cocoapods
-    assert sample_blob('Pods/blah').vendored?
```

Comment:
Can this test be move to `[test_generated()`](https://github.com/freak4pc/linguist/blob/fca6ebbcfd4101561944d387573a3b202025e650/test/test_file_blob.rb#L183`) and flipped to `.generated?`

Can a new test be added to ensure something like `Dummy/not-Pods/` is _not_ treated as generated.

Reply:
@lildude I already have these tests in generated, see here: https://github.com/github/linguist/pull/3865/files#diff-bd71790729c9298dbbdadc57b93d1038R45

As for the "not" test - I just want to confirm, we don't have these kind of test for any other of the generated folders (go/vendor, node_modules, etc). Also, not entirely sure on the test structure of that file and have some hard problem running these tests locally :) I'll try to count on CI for this.

Reply:
@lildude Would you want me to add a method to test_generated.rb along the lines of `not_generated_without_loading_data` to assert the negative of `generated_without_loading_data` ? 

Or would you want me to manually assert that test case? Since there isn't any other test case in that file to assert something _doesn't_ match

Reply:
That's not quite the same thing. The tests you've added test the Generated behaviour. The test changes I've requesting are for FileBlob.

As for the "not" tests - good point. I was thinking about trying to start testing the regexs to ensure they continue to remain accurate and limited to the scope initially intended, but as the tests stand, this will involve adding another irrelevant sample.  Ignore the "not" test for the moment.